liquid_node/commands.py

```diff
@@ -73,6 +73,8 @@ def deploy():
     for job, hcl in jobs:
         log.info('Starting %s...', job)
         nomad.run(hcl)
+        services = nomad.get_services(hcl)
+        consul.wait_for_health_checks(services)
```

Comment:
There would be some advantage to adding all services to a grand list, and checking the entire list, that way services can start in parallel.